compiler/damlc/tests/daml-test-files/SemanticsEvalOrder.daml

```diff
@@ -63,6 +64,10 @@ overApply = scenario do
   let _ = f 1 2
   pure ()
 
+errorError = scenario do
+  let _ = (error "errorError OK") (error "errorError Failed")
+  pure ()
+
```

Comment:
Isn't that exactly the same as the test below?

Reply:
Yes. Well spotted. I removed the duplicate.